api/DOMErrorHandler.json

```diff
@@ -1,100 +0,0 @@
-{
-  "api": {
-    "DOMErrorHandler": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMErrorHandler",
-        "support": {
-          "chrome": {
-            "version_added": null
-          },
-          "chrome_android": {
-            "version_added": null
-          },
-          "edge": {
-            "version_added": "≤18"
```

Comment:
I have tested Edge 18 and there is no `window.DOMErrorHandler`. I didn't expect there to be either, given the `handleError` member this was probably once a callback interface, and those aren't exposed unless they have constants.